.github/workflows/reusable-performance.yml

```
@@ -290,7 +306,7 @@ jobs:
         uses: actions/upload-artifact@b4b15b8c7c6ac21ea08fcf65892d2ee8f75cf882 # v4.4.3
         if: always()
         with:
-          name: performance-artifacts${{ inputs.memcached && '-memcached' || '' }}-${{ github.run_id }}
+          name: performance-artifacts${{ inputs.multisite && '-multisite' || '' }}${{ inputs.memcached && '-memcached' || '' }}-${{ github.run_id }}
```

Just an idea, for parity with the workflow naming:
```suggestion
          name: performance-artifacts${{ inputs.multisite && '-multisite' || '-single' }}${{ inputs.memcached && '-memcached' || '' }}-${{ github.run_id }}
```

---

Seems unnecessary for the artifact names IMO.